src/ng_package/text_replace/index.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import {globSync} from 'tinyglobby';
+
+/**
+ * Interface describing a file captured in the Bazel action.
+ * https://docs.bazel.build/versions/main/skylark/lib/File.html.
+ */
+export interface BazelFileInfo {
+  /** Execroot-relative path pointing to the file. */
+  path: string;
+  /** The path of this file relative to its root. e.g. omitting `bazel-out/<..>/bin`. */
+  shortPath: string;
+}
```

Comment:
Might be worth dropping this whole complexity here as we don't ever need a `shortPath`?